### PR TITLE
adds mlst 2.22.1

### PR DESCRIPTION
**Pull request by kapsakcj**
adds dockerfile for mlst 2.22.1. The main change here is the updated databases, code has not changed. Also updated main README.md with new mlst version and fixed typo with Freyja 1.3.8 listing.

Pull Request (PR) checklist:
- [x] Include a description of what is in this pull request in this message.
- [x] The dockerfile successfully builds to a test target for the user creating the PR. (i.e. `docker build --tag samtools:1.15test --target test docker-builds/samtools/1.15` )
- [x] Directory structure as name of the tool in lower case with special characters removed with a subdirectory of the version number (i.e. `spades/3.12.0/Dockerfile`)
   - [x] (optional) All test files are located in same directory as the Dockerfile (i.e. `shigatyper/2.0.1/test.sh`)
- [x] Create a simple container-specific [README.md](https://github.com/StaPH-B/docker-builds/blob/master/.github/workflow-templates/readme-template.md) in the same directory as the Dockerfile (i.e. `spades/3.12.0/README.md`)
   - [x] If this README is longer than 30 lines, there is an explanation as to why more detail was needed
- [x] Dockerfile includes the recommended [LABELS](https://github.com/StaPH-B/docker-builds/blob/master/dockerfile-template/Dockerfile#L8-L18) 
- [x] Main [README.md](https://github.com/StaPH-B/docker-builds/blob/master/README.md) has been updated to include the tool and/or version of the dockerfile(s) in this PR
- [x] [Program_Licenses.md](https://github.com/StaPH-B/docker-builds/blob/master/Program_Licenses.md) contains the tool(s) used in this PR and has been updated for any missing

### CODE REVIEW

File: mlst/2.22.1/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+ libmoo-perl \
+ liblist-moreutils-perl \
+ libjson-perl \
+ gzip \
+ file \
+ ncbi-blast+ && \
+ apt-get autoclean && rm -rf /var/lib/apt/lists/*
+
+# get any2fasta
+RUN cd /usr/local/bin && \
```
Comment (erinyoung) [inline mlst/2.22.1/Dockerfile:29]: Would it be a good idea to add a version to this?

Comment (kapsakcj) [reply mlst/2.22.1/Dockerfile:29]: ooo yes, that's a good idea. I'll update the dockerfile

**Comment by kapsakcj** [comment]
> My only issue is adding a version to any2fasta. I think it is 0.4.2, and not likely to updated, soon, though.

done!

**Review by erinyoung — APPROVED** [review]

**Comment by erinyoung** [comment]
Looks good! I'm going to push this to dockerhub and quay and overwrite the "latest" tag

**Pull request merged**